.github/CODEOWNERS

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# Any committer can add themselves to any  of the path patterns
+# and will subsequently get requested as a reviewer for any PRs 
+# that change matching files.
+#
+# This file uses .gitignore syntax with a few exceptions see the
+# documentation about the syntax: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# Arrow Format
+/format/
+
+## Components
+/c_glib/ @kou
+/cpp/
+/cpp/src/arrow/flight/ @lidavidm
+/csharp/
+/go/
+/java/
+/js/
+/matlab/
```

Comment:
I am not a committer. However, is there any way that I can still be a reviewer on changes to the MATLAB interface?

Reply:
Just to clarify, I am just wondering if there is any way for community members who aren't committers to be notified when certain files are changed, even if they aren't officially reviewers like committers.

Reply:
No CODEOWNERS only works for people with write access to the repo, sorry. But I would be happy to play proxy and add myself to matlab and than ping you on those :)

Reply:
```suggestion
/matlab/ @assignUser 
```

Reply:
Thanks for clarifying! That sounds great :) - thank you for volunteering to help!

Please feel free to add @lafiona, @sreeharihegden and/or myself on any of the MATLAB reviews. We are all actively working on the MATLAB code.